robot/src/main.cpp

```diff
@@ -108,6 +110,14 @@ int main(int argc, char **argv)
                 options.order_port = address.second.value();
             }
             break;
+        case 'a':
+            if (time_chosen) {
+                std::cerr << "Time system already set to system time." << std::endl;
+                exit(1);
```

Comment:
Ift future proofing tror jeg en bedre logik er bare at lade være at forbinde til en remote clock server, hvis systemtid er valgt, frem for at terminere. Det synes jeg godt -a må gøre.